app/controllers/authenticate_controller.rb

```diff
@@ -79,7 +79,7 @@ def authenticate(input = authenticator_input)
 
   def authenticate_jwt
     params[:authenticator] = "authn-jwt"
-    authn_token = Authentication::AuthnJwt::OrchestrateJwtAuthentication.new.(
+    authn_token = Authentication::AuthnJwt::OrchestrateAuthentication.new.(
```

Comment:
First, I don't like the implicit `.(` which is actually `.call(`.
Please be explicit, it helps readability.
Second, why do you name `authn_token` to the result of `OrchestrateAuthentication`?
I don't see the relation between the two, so the name looks ambiguous to me...

Reply:
Authentication::AuthnJwt::OrchestrateAuthentication.new.call